frontend/src/tests/components/Courses/CoursesForm.test.js

```diff
@@ -122,4 +145,45 @@ describe("CoursesForm tests", () => {
 
     });
 
+    //to be added for mutation
+    //line 24 in coursesform.js: [] changed to ["Stryker was here"]
+    // Added this test to address the mutation testing issue
```

Comment:
I don't think this comment really adds any value for future developers.

This is the kind of thing you can comment on in the PR if you want to document it for folks reviewing your code. 

In particular: line numbers are going to change, so as my former boss at AppFolio likes to say, this comment is a "lie waiting to happen".